src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java

```diff
@@ -209,23 +215,24 @@ public void addNotifications(ProgrammeMembership programmeMembership)
           = getLatestNotificationsSent(programmeMembership.getPersonId(),
           programmeMembership.getTisId());
 
-      createDirectNotifications(programmeMembership, notificationsAlreadySent);
+      createDirectProgrammeNotifications(programmeMembership, notificationsAlreadySent);
       createInAppNotifications(programmeMembership, notificationsAlreadySent);
     }
   }
 
   /**
-   * Create "direct" notifications, such as email, which may be scheduled for a future date/time.
+   * Create "direct" programme notifications, such as email, which may be scheduled for a future
+   * date/time.
```

Comment:
Clarified that this is specific to programmes, since we need the programme startDate to assess if its a deferral. If in future we have other notification types that need handling we can refactor this.